frontAndBackExample/models/models.js

```diff
@@ -6,13 +6,15 @@ const UserSchema = new mongoose.Schema(
 	username: {type: String, required: true},
 	password: {type: String, required: true},
 	isAdmin: {type: Boolean, default: false},
-	elo: {type: Intl, default: 0},
-	dateRegistered: {type: Date, default: Date.now}
+	elo: {type: Intl, default: 500},
+	dateRegistered: {type: Date, default: Date.now},
+	sate:{type:String}
```

Comment:
what is "sate"? Is it supposed to be "state"?

Reply:
yep.

just a thing to note, you can "request changes" using github reviews. That would have made this a much quicker thing for me to fix, cuz then I just need to "confirm change"